### PR TITLE
ci: when merging to the main branch push the latest tag image to dockerhub

### PR DESCRIPTION
**Pull request by cpanato**
**What type of PR is this?**
/kind feature

**Any specific area of the project related to this PR?**
/area build

**What this PR does / why we need it**:
When we merge to the main branch we can push the tag `latest` of `falcosidekick` image to docker hub, for people that want to live in the edge 😄 

This is not for release, just update the `lastest` tag image.

The release thing will be in a followup pr

/assign @leogr @Issif 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by cpanato** [comment]
humm, i will create a job only for master, since that uses context and the context is only available for persons that have write permission in the repo.

File: .circleci/config.yml [hunk 1]
```diff
@@ -39,10 +46,10 @@ workflows:
             tags:
               only: /[0-9]+(\.[0-9]+)*(-.*)*/
       - build-image:
+          context: falco
           requires:
             - test
             - lint
           filters:
             tags:
               only: /[0-9]+(\.[0-9]+)*(-.*)*/
```
Comment (leogr) [inline .circleci/config.yml:55]:
```suggestion
            branches:
              only: "master"
```
If we want to push the `lastest` tag only when the master has been updated, I think we have to filter just for that

**Review by leogr — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (cpanato) [reply .circleci/config.yml:55]: we also want to just to build in PRs to make sure the image can be built

Comment (leogr) [reply .circleci/config.yml:55]: So we want to build on every branch, but push the image for the master only?

**Comment by leogr** [comment]
Approved by mistake, I just wanted to comment, see https://github.com/falcosecurity/falcosidekick/pull/136#discussion_r540323043

/approve cancel

File: .circleci/config.yml (hunk 1 above)
Comment (cpanato) [reply .circleci/config.yml:60]: we go with two jobs as discussed in slack

**Review by leogr — APPROVED** [review]

**Pull request merged**